README.md

```diff
@@ -1,13 +1,86 @@
-======
-cf-plot
-======
+# cf-plot
 
-cf-plot is a set of Python plotting routines for the contour, vector and line plots that climate
-researchers commonly make.  
+## Code-light plotting for earth science and aligned research
 
+### Overview
 
-Documentation
-=============
-Please refer to the cf-plot homepage http://ajheaps.github.io/cf-plot
+**cf-plot allows you to produce and customise publication-quality contour,
+vector, line and more plots with the power of Python,
+[matplotlib](https://matplotlib.org/),
+[Cartopy](https://scitools.org.uk/cartopy/docs/latest/) and
+[cf-python](https://ncas-cms.github.io/cf-python/), in as few lines of code
+as possible.**
 
+It is designed to be a useful visualisation tool for environmental, earth
+and aligned science, for example to facilitate climate and meteorological
+research. cf-plot is developed and maintained by the
+[NCAS-CMS](https://cms.ncas.ac.uk/index.html) group, part of
+[NCAS](https://ncas.ac.uk/).
 
+
+### Brief Demonstration
+
+In as little as four lines of Python including imports, using `cf-plot` you can
+for example produce a contour plot showing a 2D subspace of a netCDF dataset:
+
+```python
+import cf
+import cfplot as cfp
+f = cf.read('<dataset name>.nc')[0]  # picks out a read-in field of the dataset
+cfp.con(f.subspace(time=<chosen time value>))  # creates a contour plot of the field at that time value
+```
+
+
+### Examples Gallery
+
+A gallery of outputs made with cf-plot, showcasing a range of plotting
+possibilities with links to relevant documentation pages and to example code,
+can be found
+[on this dedicated page within the documentation](http://ajheaps.github.io/cf-plot/gallery.html),
+as illustrated in this (static) image:
+
+![cf-plot example gallery of plots](docs/media/cf_gallery_image.png)
+
+
+### Documentation
+
+See [the cf-plot homepage](http://ajheaps.github.io/cf-plot)
+(`http://ajheaps.github.io/cf-plot`) for the full online documentation.
+
+
+### Installation
+
+To install cf-plot with its required dependencies, you can use `pip`:
+
+```bash
+pip install cf-python
+pip install cf-plot
+```
+
+or you can use `conda` (or similar package managers such
+as `mamba`) as follows (or equivalent):
+
+```bash
+conda install -c ncas -c conda-forge cf-python cf-plot udunits2
+conda install -c conda-forge mpich esmpy
+```
+
+More detail about installation is provided on the
+[installation page](http://ajheaps.github.io/cf-plot/download.html)
+(`http://ajheaps.github.io/cf-plot/download.html`)
+of the documentation.
+
+### Contributing
+
+Everyone is welcome to contribute to cf-plot, however the NCAS-CMS team reserve
+the right to only accept changes which they consider to improve the codebase
+and align with future ambitions for the library.
```

Comment:
Indeed, I just thought it might be wise to indicate that we can't be bound to accepting any old changes that someone might propose. Notably, in a previous project I worked on, someone did months of work towards a PR without checking in (formally/regularly) that it was what was wanted by the core project team, and after difficult conversations the PR was closed and not incorporated and I imagine someone felt very disgruntled. I think mainly I wanted to indicate that any work contributed should be aligned with an Issue raised and signed off by myself or someone else relevant in NCAS-CMS, but probably if we say something along those lines it should be in the more detailed Contributing guidance I will soon write up (or specifically, copy from another good OS project).

In short, I agree with your proposal but think we should add something along the above lines to the more detailed guidance, to prevent anyone possibly doing some work towards contributions which we can't accept.

Committing your suggestion (there's a typo but I'll fix that up in a final commit). Thanks.